src/components/SingleName/ShortName.js

```diff
@@ -36,20 +37,20 @@ const InnerWrapper = styled('div')`
   `}
 `
 
-export default function ShortName() {
+export default function ShortName({ name }) {
+  const label = name.split('.')[0]
+  const labelhash = `${jsSHA3.keccak256(label.toLowerCase())}`
+  const link = `https://opensea.io/assets/0xFaC7BEA255a6990f749363002136aF6556b31e04/${labelhash}`
   return (
     <ShortNameContainer>
       <InnerWrapper>
         <p>
-          Short names are not currently on auction, but they can be reserved at{' '}
-          <a href="https://reserve.ens.domains">reserve.ens.domains</a>. The
-          auctions will begin soon after the reservation process.
+          Short names are currently on auction, but they can be reserved at{' '}
+          <a href={link}>OpenSea</a>. The auctions will begin soon after the
```

Comment:
Isn't it better to open new tab?

Reply:
It also shows 404 when clicking the link to https://opensea.io/assets/0xFaC7BEA255a6990f749363002136aF6556b31e04/f06ea683845d8994338d98e7296850612317647cdb1db3d1051f24bafdefe9f7 . Does this not exist yet?

Reply:
Doesn't exist yet